oak_runtime/introspection_browser_client/components/ApplicationStateOverview/index.tsx

```diff
@@ -36,6 +36,8 @@ export default function ApplicationStateOverview({
               <dd>{id}</dd>
               <dt>Name:</dt>
               <dd>{nodeInfo.name}</dd>
+              <dt>Labels:</dt>
+              <dd>{nodeInfo.name}</dd>
```

Comment:
Sorry, missed this in my previous review. This should probably use something else rather than `nodeInfo.name`.

Reply:
Oh yes! Classic me, overlooked this. :)